### ISSUE TITLE
Domain Settings / Custom Equivalent Domains Save UI Not Clear

### ISSUE THREAD

**Issue by jyavenard** [bitwarden/browser#218]
I have a single entry in Custom Equivalent Domains
it contains:
mozilla.org, mozilla.com, auth0.com, mozilla.okta.com

information is stored under mozilla.org 

It correctly autofill web sites in the domain mozilla.org, mozilla.com or auth0.com
however, mozilla.okta.com is never filled.

Could it be because the FQN has an extra subdomain?

**Comment by mhammond** [comment]
jya and I chatted about this a little on IRC. I pointed out the domain should be just "okta.com", but I too failed to get it to work. Long story short, I got caught by suboptimal UX.

* Went to "Custom Equivalent Domains", hit "Edit", made a change and hit submit.
* Assumed that was all I needed to do - tried manually syncing, sniffed the network requests etc - everything pointed to the fact the server simply wasn't sending the new rules down.

It turns out I didn't hit the "Save" button - but nothing on the form implied there was a save pending. Once I hit "submit" and then hit "save", it started working as expected.

**Comment by jyavenard** [comment]
The UX issue mhammond mentioned also occurs when you attempt to pay for the premium subscription.

I entered my payment details, using paypal. It redirected me to paypal, I validated which got me back to the original page.
and I clicked on submit.

The next day, as I went into the settings, I realised that I wasn't a premium member yet. So I went back to the payment details and redid it all, only this time did I notice the extra save button.

Having to submit *and* save is counter intuitive.
